### PR TITLE
updated readme sounds_web

### PR DESCRIPTION
**Pull request by ankitjain0105**
Quick Changes in readme file.

### CODE REVIEW

File: config/sounds-web/README.md [hunk 1]
```diff
@@ -8,3 +8,28 @@ As well as all the normal usage options, the BBC Sounds Web tests can be pointed
...
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://some.url A11Y_CONFIG=sounds-web/all npm run start:bbc-a11y:headless
+```
+
+For lighthouse
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://some.url A11Y_CONFIG=sounds-web/all npm run start:lighthouse:headless
+```
+
+*We have also added a way to execute against particular page, the below example will run against Listen Page:
```
Comment (andymsuk) [inline config/sounds-web/README.md]:
```suggestion
We have also added a way to execute against particular page, the below example will run against Listen Page:
```

File: config/iplayer-web/README.md [hunk 2]
```diff
@@ -8,3 +8,27 @@ As well as all the normal usage options, the iPlayer Web tests can be pointed at
...
+For lighthouse
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_IPLAYER_WEB_BASE_URL=https://some.url A11Y_CONFIG=iplayer-web/all npm run start:lighthouse:headless
+```
+
+We have also added a way to execute against particular page, the below example will run against iPlayer homepage:
+
+For devs to bbc-a11y on localhost :
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_IPLAYER_WEB_BASE_URL=https://sandbox.bbc.co.uk A11Y_CONFIG=iplayer-web/app-homepage-test npm run start:bbc-a11y:headless
```
Comment (andymsuk) [inline config/iplayer-web/README.md]:
```suggestion
A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_IPLAYER_WEB_BASE_URL=https://sandbox.bbc.co.uk/iplayer A11Y_CONFIG=iplayer-web/app-homepage-test npm run start:bbc-a11y:headless
```

File: config/iplayer-web/README.md [hunk 3]
```diff
@@ -8,3 +8,27 @@ As well as all the normal usage options, the iPlayer Web tests can be pointed at
...
+```
+
+For devs to lighthouse on localhost :
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_IPLAYER_WEB_BASE_URL=https://sandbox.bbc.co.uk A11Y_CONFIG=iplayer-web/app-homepage-test npm run start:lighthouse:headless
```
Comment (andymsuk) [inline config/iplayer-web/README.md]:
```suggestion
A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_IPLAYER_WEB_BASE_URL=https://sandbox.bbc.co.uk/iplayer A11Y_CONFIG=iplayer-web/app-homepage-test npm run start:lighthouse:headless
```

File: config/sounds-web/README.md [hunk 4]
```diff
@@ -8,3 +8,27 @@ As well as all the normal usage options, the BBC Sounds Web tests can be pointed
...
+For lighthouse
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://some.url A11Y_CONFIG=sounds-web/all npm run start:lighthouse:headless
+```
+
+We have also added a way to execute against particular page, the below example will run against Listen Page:
+
+For bbc-a11y
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://www.bbc.co.uk A11Y_CONFIG=sounds-web/listen_page npm run start:bbc-a11y:headless
```
Comment (andymsuk) [inline config/sounds-web/README.md]:
```suggestion
A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://www.bbc.co.uk/sounds A11Y_CONFIG=sounds-web/listen_page npm run start:bbc-a11y:headless
```

File: config/sounds-web/README.md [hunk 5]
```diff
@@ -8,3 +8,27 @@ As well as all the normal usage options, the BBC Sounds Web tests can be pointed
...
+```
+
+For lighthouse
+```
+A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://www.bbc.co.uk A11Y_CONFIG=sounds-web/listen_page npm run start:lighthouse:headless
```
Comment (andymsuk) [inline config/sounds-web/README.md]:
```suggestion
A11Y_USERNAME=user A11Y_PASSWORD=password A11Y_SOUNDS_WEB_BASE_URL=https://www.bbc.co.uk/sounds A11Y_CONFIG=sounds-web/listen_page npm run start:lighthouse:headless
```

File: config/sounds-web/README.md (hunk 4 above)
Comment (ankitjain0105) [reply config/sounds-web/README.md]: as discussed.

File: config/sounds-web/README.md (hunk 5 above)
Comment (ankitjain0105) [reply config/sounds-web/README.md]: as discussed.

**Pull request merged**